CefSharp/DependencyChecker.cs

```diff
@@ -69,22 +69,36 @@ public static class DependencyChecker
         {
             "CefSharp.Core.dll",
             "CefSharp.dll",
-            "CefSharp.BrowserSubprocess.Core.dll",
```

Comment:
It's the `C++` wrapper around the `CEF` functionality. `BrowserSubProcess` basically exposes the functionality (using `c#` where possible, as it's easier for the most part)